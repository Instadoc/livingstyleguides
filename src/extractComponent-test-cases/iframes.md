---
name: Component Name
category: Component Category
---

This is the first specimen:

```specimen-1.html
<b>Specimen 1</b>
```

```specimen-1.css
b { color: red }
```

This is the second specimen:

```specimen-2.html hidden
<b>Specimen 2</b>
```

```specimen-2.html
<p>More specimen 2 content</p>
```

```specimen-2.css
b { color: green }
```
