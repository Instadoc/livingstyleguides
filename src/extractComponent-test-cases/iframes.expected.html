<p>This is the first specimen:</p>
<iframe src="Component Name/specimen-1.html"></iframe>
<pre><code class="language-specimen-1.html">&#x3C;b>Specimen 1&#x3C;/b>
</code></pre>
<pre><code class="language-specimen-1.css">b { color: red }
</code></pre>
<p>This is the second specimen:</p>
<iframe src="Component Name/specimen-2.html"></iframe>
<pre><code class="language-specimen-2.html">&#x3C;p>More specimen 2 content&#x3C;/p>
</code></pre>
<pre><code class="language-specimen-2.css">b { color: green }
</code></pre>